cmd/kpod/stop.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"github.com/kubernetes-incubator/cri-o/libkpod"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	defaultTimeout int64 = 10
+	stopFlags            = []cli.Flag{
+		cli.Int64Flag{
+			Name:  "timeout, t",
+			Usage: "Seconds to wait for stop before killing the container (default 10)",
+			Value: defaultTimeout,
+		},
+	}
+	stopDescription = "Stop one or more containers"
+	stopCommand     = cli.Command{
+		Name: "stop",
+		Usage: "Stops one or more running containers.  The container name or ID can be used.  A timeout to forcibly" +
```

Comment:
Take your patch and execute 
kpod stop --help

Does it look good there.

Go supports multi line strings

Usage: ` 
Stops one of m...

This should look good for the user
`

Reply:
it lgtm